tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -400,15 +400,18 @@ const reducer = createReducer(
       }
       const numExecutionsIncreased =
         numExecutions > state.executions.executionDigestsLoaded.numExecutions;
-      const newState = {
+      return {
         ...state,
         lastNonEmptyPollDataTimeMs: numExecutionsIncreased
           ? Date.now()
           : state.lastNonEmptyPollDataTimeMs,
         executions: {
           ...state.executions,
+          focusIndex:
```

Comment:
Note that there were some tricky situations for convincing the compiler that I'm properly using the types. This one took me the longest to figure out.